loginFunctions.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+session_start();
+
+$correctUsername = 'obwansan';
+$correctPassword = 'turnip100';
+
+/**
+ * Checks if username and password are correct
+ *
+ * @param $username, $password, $correctUsername, $correctPassword string
```

Comment:
need to list each param seperately 
@param (tag) $var1 (param) Int (datatype (mixed if many)) Description (eg: a num to be multiplied)
        - do this for every param that could be passed